packages/relay-compiler/codegen/__tests__/writeRelayGeneratedFile-test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails oncall+relay
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+const {
+  createPrintRequireModuleDependency,
```

Comment:
We should probably make `createPrintRequireModuleDependency` a standalone module, and import it to `writeRelayGeneratedFile`.

Reply:
This can fortunately be done in a jiffy. :-)